app/models/ex_link.rb

```diff
@@ -0,0 +1,20 @@
+class ExLink < ActiveRecord::Base
+  validates :label, :url, presence: true
+  before_save :polish_tags
+
+  has_attached_file :image,
+                    styles: { medium: "300x300>", thumb: "100x100>" },
+                    path: ":rails_root/public/system/images/ex_link/:id/:style/:filename",
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.